README.md

```diff
@@ -78,3 +96,23 @@ Please note, you're free to update this template in order to setup events publis
 |validFrom=2018-06-01T10:00+03:00
 }}
 ```
+
+### Usage of Event template
```

Comment:
Book template?:)

Reply:
That's a detail, I'll correct when all will be ok :)  I might have other things to add in ;)